libs/ui-watt/src/lib/components/checkbox/+storybook/storybook-checkbox-states.component.scss

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "@energinet-datahub/watt/utils" as watt;
+
+.mat-checkbox.error {
+  .mat-checkbox-frame {
+    border-color: red !important;
+  }
+
+  .mat-checkbox-label {
+    color: red !important;
+  }
+}
```

Comment:
is this how we want to handle errors?

I placed it in the story overview because this was just a "showcase".

In a "real" scenario I would assume that you control the error state of the checkbox from `formControls` or it being required or something to that effect.

It could add a class called `error` of course, but I think maybe we should consider a discussion on how to handle error in general on the forms elements 🙂

Reply:
Ah, I didn't realize the `error` class came from the story template. In that case, yeah it should probably get the error state via `formControls` somehow. Regardless, I think the error styling should be part of the checkbox component, we just need to figure out the best way. Perhaps you can use `formControls` in the story?

Reply:
I agree that maybe we should have "live" version of the error state on the checkbox, but I think this should be another task #1356